assets/base/i18n/fr.json

```diff
@@ -552,8 +552,8 @@
   "mobile.sidebar_settings.permanent": "Barre latérale permanente",
   "mobile.share_extension.too_long_title": "Le message est trop long",
   "mobile.share_extension.too_long_message": "Nombre de caractères : {count}/{max}",
-  "mobile.server_ssl.error.title": "Untrusted Certificate",
-  "mobile.server_ssl.error.text": "The certificate from {host} is not trusted.\n\nPlease contact your System Administrator to resolve the certificate issues and allow connections to this server.",
+  "mobile.server_ssl.error.title": "",
```

Comment:
@wget any reason why these are left empty?

Reply:
That was me, all good. Will close this PR to revert.